aggregator/src/main/java/com/vmware/flowgate/aggregator/scheduler/job/AggregatorService.java

```diff
@@ -328,15 +392,16 @@ private void syncHostTemperatureAndHumidySensor(boolean fullSync) {
          candidateServer = Arrays.asList(allServers);
       } else {
          for (Asset asset : allServers) {
-            Map<ServerSensorType, String> formulas = asset.getSensorsformulars();
-            if (formulas == null) {
-               formulas = new EnumMap<ServerSensorType, String>(ServerSensorType.class);
-               asset.setSensorsformulars(formulas);
-            }
-            if (!formulas.containsKey(ServerSensorType.BACKPANELTEMP)
-                  || !formulas.containsKey(ServerSensorType.FRONTPANELTEMP)
-                  || !formulas.containsKey(ServerSensorType.HUMIDITY)) {
+            Map<String,Map<String,Map<String,String>>> metricsFormular = asset.getMetricsformulars();
```

Comment:
this logic is not correct. 

We only try to make a server as candidate when and only when the humidity and temperature metric is null.

Reply:
Done